subsys/power/device.c

```diff
@@ -29,6 +29,13 @@ static const char *const core_devices[] = {
 	"sys_clock",
 	"UART_0",
 };
+#elif defined(CONFIG_SOC_SERIES_CC13X2_CC26X2)
+#define MAX_PM_DEVICES	15
+#define MAX_DEV_NAME_LEN	16
+static const char core_devices[][MAX_DEV_NAME_LEN] = {
```

Comment:
This should be `static const char *const core_devices[]` to match current master.